lib/renter.js

```diff
@@ -592,6 +592,17 @@ Renter.prototype._recordResponseTime = function(nodeID,
       return;
     }
 
+    // Changes spaceAvailable to false if contacts not seen in 24 hours
+    // so they stop receiving ALLOC messages
+    if (contact.lastSeen < new Date(Date.now() - 864e5)) {
+      contact.update({_id: nodeID}, {$set: {spaceAvailable: false}}, (err) => {
```

Comment:
The contact is already being saved on line 610 below this in the function, see https://github.com/nat-ventura/complex/blob/5531afedb2d119027defbe8129c8c0ddd6b22962/lib/renter.js#L610 there isn't a need to call update here.

Reply:
Okay, since the save is being called on `recordResponseTime(responseTime)` I just didn't want to overlap/overwrite with anything being saved there. So I should just call `set` inside the save function?

Reply:
You should just be able to do `contact.spaceAvailable = false`

Reply:
The mongoose model `save` function with make the corresponding mongodb query in one call.